guardian/multimedia-launchdetector-v3
# Change functionalty for next-gen system

## What does this change?
Overhauls communication to work with the next-gen multimedia production system
- Vidispine integration removed
- Targets the new pluto-deliverables component
- Authentication now using an HMAC signature
- Metadata models completely updated to the new specs

## How to test
Make sure it's pointed at the next-gen environment.  Create a media atom in CODE, wait for it to be ready and then hit Publish.  You should see a response from the CAPI stream within 30s.  This will trigger a REST request to pluto-deliverables which should update the record in the production system

DON'T MERGE AND DEPLOY TO MASTER BEFORE NEXTGEN ROLLOUT

## How can we measure success?
Still getting published masters after rollout
